streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/entity/Application.java

```diff
@@ -685,6 +691,7 @@ public static class Pom {
     private String groupId;
     private String artifactId;
     private String version;
+    private String classifier;
```

Comment:
The user's exist historical jobs. Without this classifier field, Will there be any exceptions?

Reply:
Compile with the modified version, edit a new job, upload the classifier and non-classifier packages, the classifier and non-classifier POM, and compile the package publishing task normally.